javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,48 @@
+const valueOf = 'value of';
+const typeOf = 'typeof';
+
+const width = 3;
+const height = 4;
+const area = width * height;
+
+const bill = 38;
+const payment = 50;
+const change = payment - bill;
+
+const quizzes = 85;
+const midterm = 90;
+const final = 95;
+const grade = (quizzes + midterm + final) / 3;
+
+const firstName = 'Bug';
+const lastName = 'Smith';
+const fullName = firstName + ' ' + lastName;
+
+const pH = 7;
+const isAcidic = pH <= 6.99;
+
+const headCount = 300;
+const isSparta = headCount === 300;
+
+const motto = fullName + ' ' + 'is the GOAT';
```

Comment:
`motto` should be initialized with a value of `firstName` originally, and then you should use the `+=` operator to reassign a new value to the variable. So this should be broken up into 2 lines, rather than doing this all here when you initialize it